src/backend/renderer/multigpu/mod.rs

```diff
@@ -1186,6 +1185,8 @@ where
                     copy_rects = Vec::from([Rectangle::from_loc_and_size((0, 0), buffer_size)]);
                 }
 
+                sync.wait();
```

Comment:
On the other hand we should be able to do a `frame.wait(&sync)` here instead, no?
(Although OpenGL will also synchronize this correctly internally.)

Reply:
I added this to make sure the frame has finished before calling copy_framebuffer. So I don't think waiting in the target frame would work. In case of gl this is probably not needed, yeah.

Reply:
Given we only have the GLRenderer and the API currently expects implicit sync, I would not bother explicitly waiting here. I expect a discussion on copy_framebuffer and other apis to come up, when implementing the vulkan renderer.

Also I didn't meant `target_frame.wait`, but `render.renderer().wait`, I didn't realize this was an API of `Frame`.

Reply:
removed the explicit wait here